### DISCUSSION TITLE
Floating sticky notes - VSCode

### DISCUSSION THREAD

**Discussion by luisafernandarj** [Hyperclaw79/vssticky#9] [Ideas]
Hi,
I have an idea to add sticky notes inside VSCode which will be like a floating window, and those sticky notes stay in that code where it was located.. not only giving a general note in a doc, but also a specific one.

**Comment by Hyperclaw79** [comment]
> Hi,
> I have an idea to add sticky notes inside VSCode which will be like a floating window, and those sticky notes stay in that code where it was located.. not only giving a general note in a doc, but also a specific one.

That was my exact plan initially. 😅
However, after checking out the VSCode API and its limitations, I couldn't find a way to do it at the time of development. There's no way to overlay a window on top of the editor. While it might be possible to add basic greyed out annotations next to a code line, it wouldn't be suitable for a sticky note.

If you do have a working solution, feel free to open a PR.
